README.md

```diff
@@ -1 +1,24 @@
-# Blog-app
\ No newline at end of file
+# README
```

Comment:
- **[OPTIONAL]** I would recommend you edit your readme docs using the microverse template